### PR TITLE
New Post UI Responsive + New Component Created for New Post

### PR DESCRIPTION
**Pull request by Gurmeet-Singh657**
Issue Number 476

New Post UI was made Responsive for Mobiles as well. Here it looks like
image

After Adding the post, looks like
image

And Also, the New Post component is created separately.
Please merge my pull request as a GSSOC'23 Contributor.

### CODE REVIEW

**Comment by narayan954** [comment]
Hi @Gurmeet-Singh657 , this vertical view would look good in mobile devices and current view is better for desktops actually, I think, what do you say? I'm yet to look into this pr. Do you think you can handle these separate view for desktop and mobiles?

File: src/components/NewPost.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { auth } from "../lib/firebase";
+import { useState } from "react";
+import ImgUpload from "../components/ImgUpload";
+import { AiOutlineClose } from "react-icons/ai";
+import { Dialog } from "@mui/material";
+
+export default function NewPost({ openNewUpload, setOpenNewUpload }) {
+    const user = auth.currentUser;
+    console.log("I am Happy Today");
```
Comment (narayan954) [inline src/components/NewPost.jsx]: no console logs in production

**Comment by narayan954** [comment]
Can you please look into this?? 
image

File: src/components/NewPost.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { AiOutlineClose } from "react-icons/ai";
+import { Dialog } from "@mui/material";
+import ImgUpload from "../components/ImgUpload";
+import React from "react";
+import { auth } from "../lib/firebase";
+
+export default function NewPost({ openNewUpload, setOpenNewUpload }) {
+  const user = auth.currentUser;
+  console.log("I am Happy Today");
```
Comment (narayan954) [inline src/components/NewPost.jsx:9]: no console logs in production

**Review by narayan954 — CHANGES_REQUESTED** [review]
mobile screens are not responsive, yet

**Comment by narayan954** [comment]
> Issue Number 476
> 
> New Post UI was made Responsive for Mobiles as well. Here it looks like image
> 
> After Adding the post, looks like image
> 
> And Also, the New Post component is created separately. Please merge my pull request as a GSSOC'23 Contributor.

this is tablet view/screen

**Comment by Gurmeet-Singh657** [comment]
Ohk, I will fix it 🙂

**Comment by narayan954** [comment]
closing this, you can open another pr

**Pull request closed without merging**